addons/packages/antrea/0.11.3/bundle/config/schema.yaml

```diff
@@ -31,3 +31,12 @@ antrea:
       AntreaTraceflow: false
       #@schema/desc "Flag to enable/disable network policy stats"
       NetworkPolicyStats: false
+  image:
+    #@schema/desc "The repository of antrea image"
```

Comment:
I think these added fields should be nullable.

Reply:
I am following the same logic as vsphere-cpi